scripts/exobsmon_plot.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#------------------
+# exobsmon_plot.sh
+#------------------
+
+#---------------------------
+# Generate requested plots
+#
+python ${USHobsmon}/plotObsMon.py -i ${PARMobsmon}/${MODEL}/gfs_plots.yaml  -p ${PDATE}
```

Comment:
may want to make `python` into something like `APRUN_PY`? just so we can control it in case we need to provide the full path to the interpreter?

Reply:
Fair point.  I'll make that change directly.

Reply:
Done.